prov/verbs/src/verbs_info.c

```diff
@@ -1916,14 +1914,18 @@ int vrb_getinfo(uint32_t version, const char *node, const char *service,
 {
 	int ret;
 
+	ofi_mutex_lock(&vrb_info_mutex);
```

Comment:
Can you move this after vrb_prof_func_start() to avoid including that call within the mutex?  I want to avoid someone thinking the mutex is needed for it.